lib/config/fileconf.go

```diff
@@ -834,6 +837,25 @@ type Kube struct {
 	ClusterName string `yaml:"cluster_name,omitempty"`
 }
 
+// Kube is a `kubernetes_service`
+type Kube struct {
+	// Service is a generic service configuration section
+	Service `yaml:",inline"`
+	// PublicAddr is a publicly advertised address of the kubernetes service
+	PublicAddr utils.Strings `yaml:"public_addr,omitempty"`
+	// KubeconfigFile is an optional path to kubeconfig file,
+	// if specified, teleport will use API server address and
+	// trusted certificate authority information from it
+	KubeconfigFile string `yaml:"kubeconfig_file,omitempty"`
+	// KubeClusterName is the name of a kubernetes cluster this service is
+	// running in. If set, this proxy will handle kubernetes requests for the
+	// cluster.
+	KubeClusterName string `yaml:"kube_cluster_name,omitempty"`
+	// Labels for RBAC on kubernetes clusters.
+	Labels   map[string]string `yaml:"labels,omitempty"`
+	Commands []CommandLabel    `yaml:"commands,omitempty"`
```

Comment:
I think I'm calling these `StaticLabels` and `DynamicLabels` so we refer to them in code the same as we do in documentation.

Reply:
Renamed.
We may want to rename these for nodes too at some point, I was copying from there.